Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/cart/Cart.kt

```diff
@@ -150,12 +160,115 @@ private fun CartContent(
             )
         }
         items(orderLines) { orderLine ->
-            CartItem(
-                orderLine = orderLine,
-                removeSnack = removeSnack,
-                increaseItemCount = increaseItemCount,
-                decreaseItemCount = decreaseItemCount,
-                onSnackClick = onSnackClick
+            SwipeDismissItem(
+                background = { offsetX ->
+                    // Adjust padding right for the main Row, while the delete looks like an Icon the padding must be different from zero
+                    val paddingRight = if (offsetX < -15.dp && offsetX > -200.dp) {
```

Comment:
These values should align to a 8.dp grid

Reply:
Padding right no longer exist, but kept other values align with 8.dp grid